frontend/packages/integration-tests-cypress/views/details-page.ts

```diff
@@ -0,0 +1,15 @@
+export const detailsPage = {
+  titleShouldBe: (title: string) => cy.byTestID('resource-title').contains(title), // long timeout for detail page to load,
```

Comment:
I'm confusing by the "long timeout" comment. I don't see a timeout.

Should this be called `titleShouldContain`?

Reply:
fixed.  Removed obsolete comment and renamed to `titleShouldContain`